src/components/expandable-row.vue

```diff
@@ -73,25 +73,15 @@ export default {
   }
 
   .button-cell {
-    font-size: 20px;
-    text-align: right;
-    line-height: normal;
+    align-self: center;
+    text-align: center;
     padding: 0px;
+    width: 30px;
 
-    a {
-      color: #888;
```

Comment:
It looks like this style is currently overridden. On the QA server, the link's color is `#666`.